components/src/components/dropdown/dropdown-option.tsx

```diff
@@ -54,10 +57,7 @@ export class DropdownOption {
       });
     }
   }
-  @Listen('click')
-  handleClick(ev) {
-    ev.stopPropagation();
-  }
+
```

Comment:
```suggestion
  @Listen('click')
  handleSelection(ev) {
    if(this.isMultiSelectOption){
      ev.stopPropagation()
    }
  }
```
Maybe just add it on the multiselect for now?